README.md

```diff
@@ -25,37 +23,69 @@ Then start the server with gulp.
 gulp
 ```
 
+You can also use gulp with a local .env file for environmental variables.
+
+```
+npm run start-local
+```
+
+See [package.json](package.json)'s `scripts` section for more details.
+
+### Local .env file configurations
+
+Using a local .env file, you can set environmental vairables to emulate
```

Comment:
"vairables" should be spelled "variables"

Reply:
Also, instead of "emulating" different running conditions, these settings actually would change the running conditions, right?

Reply:
Changed to "to change the running conditions"